modules/install-osp-deploy-dualstack.adoc

```diff
@@ -56,30 +55,29 @@ networking:
     hostPrefix: 23
   - cidr: fd01::/48
     hostPrefix: 64
-  serviceNetwork: <2>
+  serviceNetwork: <1>
```

Comment:
shall we sort the note by number, right now <2> (machineNetwork and clusterNetwork) is before <1>？